frontend/public/components/utils/resource-log.jsx

```diff
@@ -53,21 +64,24 @@ export class ResourceLog extends SafetyFirst {
     super(props);
     this._buffer = [];
     this._download = this._download.bind(this);
+    this._toggleFullscreen = this._toggleFullscreen.bind(this);
     this._onClose = this._onClose.bind(this);
     this._onError = this._onError.bind(this);
     this._onMessage = this._onMessage.bind(this);
     this._onOpen = this._onOpen.bind(this);
     this._restartStream = this._restartStream.bind(this);
     this._toggleStreaming = this._toggleStreaming.bind(this);
     this._updateStatus = this._updateStatus.bind(this);
+    this._logConsole = React.createRef();
```

Comment:
Could do `this._resourceLogRef = React.createRef();` or something along those lines?